factory/ethElrondBridgeComponents.go

```diff
@@ -183,7 +188,7 @@ func (components *ethElrondBridgeComponents) createElrondClient(args ArgsEthereu
 	clientArgs := elrond.ClientArgs{
 		GasMapConfig:                 elrondConfigs.GasMap,
 		Proxy:                        args.Proxy,
-		Log:                          logger.GetOrCreate("elrond client"), // TODO use loggerWithIdentifier
+		Log:                          core.NewLoggerWithIdentifier(logger.GetOrCreate("elrond client"), elrondClientLogId),
```

Comment:
we can have something like: 
```GO
core.NewLoggerWithIdentifier(logger.GetOrCreate(elrondClientLogId), elrondClientLogId),
```

in order to avoid hardcoded strings. Valid for all occurrences.